Gemfile

```diff
@@ -1,27 +1,29 @@
 source 'https://rubygems.org'
 
-ruby "2.7.8"
+ruby "3.3.8"
 
-gem 'rails', '~> 6.1.0.0'
+gem "rails", "~> 6.1.7"
 
 gem 'sprockets-rails', '~> 3.2.2'
 gem 'mysql2', '~> 0.5.0'
 gem 'bootstrap-sass'
-gem 'sass-rails', '~> 6.0.0'
+#gem 'sass-rails', '~> 6.0.0'
+gem "sassc-rails", ">= 2.1.2"
+
 gem 'uglifier'
 gem 'coffee-rails'
 gem "feature_flipper"
 
 gem 'jquery-rails'
 gem 'jbuilder', '~> 2.0'
-gem 'sdoc', '~> 0.4.0', group: :doc
+#gem 'sdoc', '~> 0.4.0', group: :doc
 
 gem 'devise', '~> 4.7.3'
 gem 'neat', '1.7.1'
 gem 'bourbon', '~> 4.2.2'
 gem 'draper'
-gem 'paperclip', '~> 6.0.0'
-gem 'aws-sdk-s3', '~> 1.98.0'
+gem "kt-paperclip", "~> 6.4", ">= 6.4.1"
```

Comment:
This is the supported version of paperclip now that thoughtbot no longer maintains.